src/qibolab/transpilers/placer.py

```diff
@@ -120,23 +127,29 @@ def __call__(self, circuit: Circuit):
 
         Args:
             circuit (qibo.models.Circuit): circuit to be transpiled.
+
+        Returns:
+            (dict): physical to logical qubit mapping.
         """
-        circuit_repr = create_circuit_repr(circuit)
-        if len(circuit_repr) < 3:
+        gates_qubis_pairs = find_gates_qubits_pairs(circuit)
```

Comment:
If you prefer this way, do whatever you wish, you'll be responsible to maintain this code anyhow.

But can you correct the typo?
```suggestion
        gates_qubits_pairs = find_gates_qubits_pairs(circuit)
```